client/lib/signup/step-actions/index.js

```diff
@@ -488,10 +487,9 @@ function processItemCart(
 	}
 }
 
-function prepareItemForAddingToCart( item, state ) {
-	const productsList = getProductsList( state );
+function prepareItemForAddingToCart( item ) {
```

Comment:
Note that now that we've removed `state` argument from the function, we should also remove the second `reduxState` arg from  the 3 invocations of `prepareItemForAddingToCart` above in this file.

Reply:
Good eye. I missed that. https://github.com/Automattic/wp-calypso/pull/59472 will get it

Reply:
Thanks for addressing it so swiftly 🚀